queries.yaml

```diff
@@ -432,6 +432,23 @@ post_process:
         - transit
         - water
 
+  # drop the names of small landuse polygons
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: landuse
+      start_zoom: 9
+      end_zoom: 12
+      properties: [name, operator, sport]
+      where: >-
+        pixel_area > area
+      geom_types: [Polygon, MultiPolygon]
```

Comment:
Looks like this will drop these properties, even if for some reason they cannot be merged later on. Is that what we want to do?